grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
Wait, scratch that suggestion. That's only gonna make things even more complex, because care will need to be taken to stop `source.js` from consuming the closing quote-character.

Is there any logical, **practical** reasoning for this, other than the old *"it seems too hacky to me"*...? Heck, for all you know, you could be ditching this grammar in six months in favour of Tree-Sitter or Bush-Croucher or something... =)